modules/hostap/src/supp_api.c

```diff
@@ -1365,6 +1365,15 @@ static int wpas_disconnect_network(const struct device *dev, int cur_mode)
 		wifi_mgmt_raise_disconnect_complete_event(iface, ret);
 	}
 
+#ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE
```

Comment:
maybe should also add it in `wpas_add_and_config_network`? As we can connect again without disconnecting.

Reply:
Yes, should be changed like this, updated as comment.